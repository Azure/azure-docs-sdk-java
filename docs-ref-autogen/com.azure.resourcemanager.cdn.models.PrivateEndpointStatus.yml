### YamlMime:JavaType
uid: "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus"
fullName: "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus"
name: "PrivateEndpointStatus"
nameWithType: "PrivateEndpointStatus"
summary: "Defines values for PrivateEndpointStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointStatus extends ExpandableStringEnum<PrivateEndpointStatus>"
constructors:
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.PrivateEndpointStatus()"
fields:
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.APPROVED"
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.DISCONNECTED"
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.PENDING"
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.REJECTED"
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.TIMEOUT"
methods:
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.cdn.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-cdn:2.1.0
