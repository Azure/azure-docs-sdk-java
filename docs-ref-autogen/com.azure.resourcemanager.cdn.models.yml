### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.cdn.models"
fullName: "com.azure.resourcemanager.cdn.models"
name: "com.azure.resourcemanager.cdn.models"
summary: "Package containing the data models for CdnManagementClient."
classes:
- "com.azure.resourcemanager.cdn.models.ActionType"
- "com.azure.resourcemanager.cdn.models.ActivatedResourceReference"
- "com.azure.resourcemanager.cdn.models.AfdCertificateType"
- "com.azure.resourcemanager.cdn.models.AfdDomainHttpsParameters"
- "com.azure.resourcemanager.cdn.models.AfdDomainListResult"
- "com.azure.resourcemanager.cdn.models.AfdDomainUpdateParameters"
- "com.azure.resourcemanager.cdn.models.AfdEndpointListResult"
- "com.azure.resourcemanager.cdn.models.AfdEndpointProtocols"
- "com.azure.resourcemanager.cdn.models.AfdEndpointUpdateParameters"
- "com.azure.resourcemanager.cdn.models.AfdOriginGroupListResult"
- "com.azure.resourcemanager.cdn.models.AfdOriginGroupUpdateParameters"
- "com.azure.resourcemanager.cdn.models.AfdOriginListResult"
- "com.azure.resourcemanager.cdn.models.AfdOriginUpdateParameters"
- "com.azure.resourcemanager.cdn.models.AfdProvisioningState"
- "com.azure.resourcemanager.cdn.models.AfdPurgeParameters"
- "com.azure.resourcemanager.cdn.models.AfdQueryStringCachingBehavior"
- "com.azure.resourcemanager.cdn.models.AfdRouteCacheConfiguration"
- "com.azure.resourcemanager.cdn.models.AfdStateProperties"
- "com.azure.resourcemanager.cdn.models.Algorithm"
- "com.azure.resourcemanager.cdn.models.AutoGeneratedDomainNameLabelScope"
- "com.azure.resourcemanager.cdn.models.AzureFirstPartyManagedCertificate"
- "com.azure.resourcemanager.cdn.models.AzureFirstPartyManagedCertificateParameters"
- "com.azure.resourcemanager.cdn.models.CacheBehavior"
- "com.azure.resourcemanager.cdn.models.CacheConfiguration"
- "com.azure.resourcemanager.cdn.models.CacheExpirationActionParameters"
- "com.azure.resourcemanager.cdn.models.CacheKeyQueryStringActionParameters"
- "com.azure.resourcemanager.cdn.models.CacheType"
- "com.azure.resourcemanager.cdn.models.CdnCertificateSourceParameters"
- "com.azure.resourcemanager.cdn.models.CdnManagedHttpsParameters"
- "com.azure.resourcemanager.cdn.models.CdnWebApplicationFirewallPolicyList"
- "com.azure.resourcemanager.cdn.models.CdnWebApplicationFirewallPolicyPatchParameters"
- "com.azure.resourcemanager.cdn.models.Certificate"
- "com.azure.resourcemanager.cdn.models.CertificateSource"
- "com.azure.resourcemanager.cdn.models.CertificateType"
- "com.azure.resourcemanager.cdn.models.CheckEndpointNameAvailabilityInput"
- "com.azure.resourcemanager.cdn.models.CheckHostnameAvailabilityInput"
- "com.azure.resourcemanager.cdn.models.CheckNameAvailabilityInput"
- "com.azure.resourcemanager.cdn.models.CheckNameAvailabilityResult"
- "com.azure.resourcemanager.cdn.models.CidrIpAddress"
- "com.azure.resourcemanager.cdn.models.ClientPortMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.ClientPortOperator"
- "com.azure.resourcemanager.cdn.models.CompressionSettings"
- "com.azure.resourcemanager.cdn.models.ContinentsResponseContinentsItem"
- "com.azure.resourcemanager.cdn.models.ContinentsResponseCountryOrRegionsItem"
- "com.azure.resourcemanager.cdn.models.CookiesMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.CookiesOperator"
- "com.azure.resourcemanager.cdn.models.CustomDomainHttpsParameters"
- "com.azure.resourcemanager.cdn.models.CustomDomainListResult"
- "com.azure.resourcemanager.cdn.models.CustomDomainParameters"
- "com.azure.resourcemanager.cdn.models.CustomDomainResourceState"
- "com.azure.resourcemanager.cdn.models.CustomDomainValidationResult"
- "com.azure.resourcemanager.cdn.models.CustomHttpsProvisioningState"
- "com.azure.resourcemanager.cdn.models.CustomHttpsProvisioningSubstate"
- "com.azure.resourcemanager.cdn.models.CustomRule"
- "com.azure.resourcemanager.cdn.models.CustomRuleEnabledState"
- "com.azure.resourcemanager.cdn.models.CustomRuleList"
- "com.azure.resourcemanager.cdn.models.CustomerCertificate"
- "com.azure.resourcemanager.cdn.models.CustomerCertificateParameters"
- "com.azure.resourcemanager.cdn.models.DeepCreatedCustomDomain"
- "com.azure.resourcemanager.cdn.models.DeepCreatedOrigin"
- "com.azure.resourcemanager.cdn.models.DeepCreatedOriginGroup"
- "com.azure.resourcemanager.cdn.models.DeleteRule"
- "com.azure.resourcemanager.cdn.models.DeliveryRule"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleAction"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleActionValue"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleCacheExpirationAction"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleCacheKeyQueryStringAction"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleClientPortCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleCookiesCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleHostnameCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleHttpVersionCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleIsDeviceCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRulePostArgsCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleQueryStringCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRemoteAddressCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRequestBodyCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRequestHeaderAction"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRequestHeaderCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRequestMethodCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRequestSchemeCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRequestUriCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleResponseHeaderAction"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleRouteConfigurationOverrideAction"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleServerPortCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleSocketAddrCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleSslProtocolCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleUrlFileExtensionCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleUrlFileNameCondition"
- "com.azure.resourcemanager.cdn.models.DeliveryRuleUrlPathCondition"
- "com.azure.resourcemanager.cdn.models.DeploymentStatus"
- "com.azure.resourcemanager.cdn.models.DestinationProtocol"
- "com.azure.resourcemanager.cdn.models.DimensionProperties"
- "com.azure.resourcemanager.cdn.models.DomainValidationProperties"
- "com.azure.resourcemanager.cdn.models.DomainValidationState"
- "com.azure.resourcemanager.cdn.models.EdgeNode"
- "com.azure.resourcemanager.cdn.models.EdgenodeResult"
- "com.azure.resourcemanager.cdn.models.EnabledState"
- "com.azure.resourcemanager.cdn.models.EndpointListResult"
- "com.azure.resourcemanager.cdn.models.EndpointPropertiesUpdateParametersDeliveryPolicy"
- "com.azure.resourcemanager.cdn.models.EndpointPropertiesUpdateParametersWebApplicationFirewallPolicyLink"
- "com.azure.resourcemanager.cdn.models.EndpointProvisioningState"
- "com.azure.resourcemanager.cdn.models.EndpointResource"
- "com.azure.resourcemanager.cdn.models.EndpointResourceState"
- "com.azure.resourcemanager.cdn.models.EndpointUpdateParameters"
- "com.azure.resourcemanager.cdn.models.ForwardingProtocol"
- "com.azure.resourcemanager.cdn.models.GeoFilter"
- "com.azure.resourcemanager.cdn.models.HeaderAction"
- "com.azure.resourcemanager.cdn.models.HeaderActionParameters"
- "com.azure.resourcemanager.cdn.models.HealthProbeParameters"
- "com.azure.resourcemanager.cdn.models.HostnameMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.HostnameOperator"
- "com.azure.resourcemanager.cdn.models.HttpErrorRangeParameters"
- "com.azure.resourcemanager.cdn.models.HttpVersionMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.HttpVersionOperator"
- "com.azure.resourcemanager.cdn.models.HttpsRedirect"
- "com.azure.resourcemanager.cdn.models.IpAddressGroup"
- "com.azure.resourcemanager.cdn.models.IsDeviceMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.IsDeviceMatchConditionParametersMatchValuesItem"
- "com.azure.resourcemanager.cdn.models.IsDeviceOperator"
- "com.azure.resourcemanager.cdn.models.KeyVaultCertificateSourceParameters"
- "com.azure.resourcemanager.cdn.models.KeyVaultSigningKeyParameters"
- "com.azure.resourcemanager.cdn.models.LinkToDefaultDomain"
- "com.azure.resourcemanager.cdn.models.LoadBalancingSettingsParameters"
- "com.azure.resourcemanager.cdn.models.LoadParameters"
- "com.azure.resourcemanager.cdn.models.LogMetric"
- "com.azure.resourcemanager.cdn.models.LogMetricsGranularity"
- "com.azure.resourcemanager.cdn.models.LogMetricsGroupBy"
- "com.azure.resourcemanager.cdn.models.LogRanking"
- "com.azure.resourcemanager.cdn.models.LogRankingMetric"
- "com.azure.resourcemanager.cdn.models.LogSpecification"
- "com.azure.resourcemanager.cdn.models.ManagedCertificate"
- "com.azure.resourcemanager.cdn.models.ManagedCertificateParameters"
- "com.azure.resourcemanager.cdn.models.ManagedRuleDefinition"
- "com.azure.resourcemanager.cdn.models.ManagedRuleEnabledState"
- "com.azure.resourcemanager.cdn.models.ManagedRuleGroupDefinition"
- "com.azure.resourcemanager.cdn.models.ManagedRuleGroupOverride"
- "com.azure.resourcemanager.cdn.models.ManagedRuleOverride"
- "com.azure.resourcemanager.cdn.models.ManagedRuleSet"
- "com.azure.resourcemanager.cdn.models.ManagedRuleSetDefinitionList"
- "com.azure.resourcemanager.cdn.models.ManagedRuleSetList"
- "com.azure.resourcemanager.cdn.models.MatchCondition"
- "com.azure.resourcemanager.cdn.models.MatchProcessingBehavior"
- "com.azure.resourcemanager.cdn.models.MatchVariable"
- "com.azure.resourcemanager.cdn.models.MetricAvailability"
- "com.azure.resourcemanager.cdn.models.MetricSpecification"
- "com.azure.resourcemanager.cdn.models.MetricsResponseGranularity"
- "com.azure.resourcemanager.cdn.models.MetricsResponseSeriesItem"
- "com.azure.resourcemanager.cdn.models.MetricsResponseSeriesItemData"
- "com.azure.resourcemanager.cdn.models.MetricsResponseSeriesItemUnit"
- "com.azure.resourcemanager.cdn.models.MetricsResponseSeriesPropertiesItemsItem"
- "com.azure.resourcemanager.cdn.models.Operation"
- "com.azure.resourcemanager.cdn.models.OperationDisplay"
- "com.azure.resourcemanager.cdn.models.OperationsListResult"
- "com.azure.resourcemanager.cdn.models.Operator"
- "com.azure.resourcemanager.cdn.models.OptimizationType"
- "com.azure.resourcemanager.cdn.models.OriginGroupListResult"
- "com.azure.resourcemanager.cdn.models.OriginGroupOverride"
- "com.azure.resourcemanager.cdn.models.OriginGroupOverrideAction"
- "com.azure.resourcemanager.cdn.models.OriginGroupOverrideActionParameters"
- "com.azure.resourcemanager.cdn.models.OriginGroupProvisioningState"
- "com.azure.resourcemanager.cdn.models.OriginGroupResourceState"
- "com.azure.resourcemanager.cdn.models.OriginGroupUpdateParameters"
- "com.azure.resourcemanager.cdn.models.OriginListResult"
- "com.azure.resourcemanager.cdn.models.OriginProvisioningState"
- "com.azure.resourcemanager.cdn.models.OriginResourceState"
- "com.azure.resourcemanager.cdn.models.OriginUpdateParameters"
- "com.azure.resourcemanager.cdn.models.ParamIndicator"
- "com.azure.resourcemanager.cdn.models.PolicyEnabledState"
- "com.azure.resourcemanager.cdn.models.PolicyMode"
- "com.azure.resourcemanager.cdn.models.PolicyResourceState"
- "com.azure.resourcemanager.cdn.models.PolicySettings"
- "com.azure.resourcemanager.cdn.models.PolicySettingsDefaultCustomBlockResponseStatusCode"
- "com.azure.resourcemanager.cdn.models.PostArgsMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.PostArgsOperator"
- "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus"
- "com.azure.resourcemanager.cdn.models.ProfileListResult"
- "com.azure.resourcemanager.cdn.models.ProfileProvisioningState"
- "com.azure.resourcemanager.cdn.models.ProfileResourceState"
- "com.azure.resourcemanager.cdn.models.ProfileUpdateParameters"
- "com.azure.resourcemanager.cdn.models.ProtocolType"
- "com.azure.resourcemanager.cdn.models.ProvisioningState"
- "com.azure.resourcemanager.cdn.models.PurgeParameters"
- "com.azure.resourcemanager.cdn.models.QueryStringBehavior"
- "com.azure.resourcemanager.cdn.models.QueryStringMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.QueryStringOperator"
- "com.azure.resourcemanager.cdn.models.RankingsResponseTablesItem"
- "com.azure.resourcemanager.cdn.models.RankingsResponseTablesPropertiesItemsItem"
- "com.azure.resourcemanager.cdn.models.RankingsResponseTablesPropertiesItemsMetricsItem"
- "com.azure.resourcemanager.cdn.models.RateLimitRule"
- "com.azure.resourcemanager.cdn.models.RateLimitRuleList"
- "com.azure.resourcemanager.cdn.models.RedirectType"
- "com.azure.resourcemanager.cdn.models.RemoteAddressMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.RemoteAddressOperator"
- "com.azure.resourcemanager.cdn.models.RequestBodyMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.RequestBodyOperator"
- "com.azure.resourcemanager.cdn.models.RequestHeaderMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.RequestHeaderOperator"
- "com.azure.resourcemanager.cdn.models.RequestMethodMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.RequestMethodMatchConditionParametersMatchValuesItem"
- "com.azure.resourcemanager.cdn.models.RequestMethodOperator"
- "com.azure.resourcemanager.cdn.models.RequestSchemeMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.RequestSchemeMatchConditionParametersMatchValuesItem"
- "com.azure.resourcemanager.cdn.models.RequestUriMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.RequestUriOperator"
- "com.azure.resourcemanager.cdn.models.ResourceReference"
- "com.azure.resourcemanager.cdn.models.ResourceType"
- "com.azure.resourcemanager.cdn.models.ResourceUsage"
- "com.azure.resourcemanager.cdn.models.ResourceUsageListResult"
- "com.azure.resourcemanager.cdn.models.ResourceUsageUnit"
- "com.azure.resourcemanager.cdn.models.ResourcesResponseCustomDomainsItem"
- "com.azure.resourcemanager.cdn.models.ResourcesResponseEndpointsItem"
- "com.azure.resourcemanager.cdn.models.ResourcesResponseEndpointsPropertiesItemsItem"
- "com.azure.resourcemanager.cdn.models.ResponseBasedOriginErrorDetectionParameters"
- "com.azure.resourcemanager.cdn.models.RouteConfigurationOverrideActionParameters"
- "com.azure.resourcemanager.cdn.models.RouteListResult"
- "com.azure.resourcemanager.cdn.models.RouteUpdateParameters"
- "com.azure.resourcemanager.cdn.models.RuleCacheBehavior"
- "com.azure.resourcemanager.cdn.models.RuleIsCompressionEnabled"
- "com.azure.resourcemanager.cdn.models.RuleListResult"
- "com.azure.resourcemanager.cdn.models.RuleQueryStringCachingBehavior"
- "com.azure.resourcemanager.cdn.models.RuleSetListResult"
- "com.azure.resourcemanager.cdn.models.RuleUpdateParameters"
- "com.azure.resourcemanager.cdn.models.SecretListResult"
- "com.azure.resourcemanager.cdn.models.SecretParameters"
- "com.azure.resourcemanager.cdn.models.SecretType"
- "com.azure.resourcemanager.cdn.models.SecurityPolicyListResult"
- "com.azure.resourcemanager.cdn.models.SecurityPolicyPropertiesParameters"
- "com.azure.resourcemanager.cdn.models.SecurityPolicyType"
- "com.azure.resourcemanager.cdn.models.SecurityPolicyUpdateParameters"
- "com.azure.resourcemanager.cdn.models.SecurityPolicyWebApplicationFirewallAssociation"
- "com.azure.resourcemanager.cdn.models.SecurityPolicyWebApplicationFirewallParameters"
- "com.azure.resourcemanager.cdn.models.ServerPortMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.ServerPortOperator"
- "com.azure.resourcemanager.cdn.models.ServiceSpecification"
- "com.azure.resourcemanager.cdn.models.SharedPrivateLinkResourceProperties"
- "com.azure.resourcemanager.cdn.models.Sku"
- "com.azure.resourcemanager.cdn.models.SkuName"
- "com.azure.resourcemanager.cdn.models.SocketAddrMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.SocketAddrOperator"
- "com.azure.resourcemanager.cdn.models.SslProtocol"
- "com.azure.resourcemanager.cdn.models.SslProtocolMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.SslProtocolOperator"
- "com.azure.resourcemanager.cdn.models.Status"
- "com.azure.resourcemanager.cdn.models.Transform"
- "com.azure.resourcemanager.cdn.models.TransformType"
- "com.azure.resourcemanager.cdn.models.UpdateRule"
- "com.azure.resourcemanager.cdn.models.UrlFileExtensionMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.UrlFileExtensionOperator"
- "com.azure.resourcemanager.cdn.models.UrlFileNameMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.UrlFileNameOperator"
- "com.azure.resourcemanager.cdn.models.UrlPathMatchConditionParameters"
- "com.azure.resourcemanager.cdn.models.UrlPathOperator"
- "com.azure.resourcemanager.cdn.models.UrlRedirectAction"
- "com.azure.resourcemanager.cdn.models.UrlRedirectActionParameters"
- "com.azure.resourcemanager.cdn.models.UrlRewriteAction"
- "com.azure.resourcemanager.cdn.models.UrlRewriteActionParameters"
- "com.azure.resourcemanager.cdn.models.UrlSigningAction"
- "com.azure.resourcemanager.cdn.models.UrlSigningActionParameters"
- "com.azure.resourcemanager.cdn.models.UrlSigningKey"
- "com.azure.resourcemanager.cdn.models.UrlSigningKeyParameters"
- "com.azure.resourcemanager.cdn.models.UrlSigningParamIdentifier"
- "com.azure.resourcemanager.cdn.models.UsageName"
- "com.azure.resourcemanager.cdn.models.UsageUnit"
- "com.azure.resourcemanager.cdn.models.UsagesListResult"
- "com.azure.resourcemanager.cdn.models.UserManagedHttpsParameters"
- "com.azure.resourcemanager.cdn.models.ValidateCustomDomainInput"
- "com.azure.resourcemanager.cdn.models.ValidateProbeInput"
- "com.azure.resourcemanager.cdn.models.ValidateSecretInput"
- "com.azure.resourcemanager.cdn.models.WafAction"
- "com.azure.resourcemanager.cdn.models.WafGranularity"
- "com.azure.resourcemanager.cdn.models.WafMatchVariable"
- "com.azure.resourcemanager.cdn.models.WafMetric"
- "com.azure.resourcemanager.cdn.models.WafMetricsResponseGranularity"
- "com.azure.resourcemanager.cdn.models.WafMetricsResponseSeriesItem"
- "com.azure.resourcemanager.cdn.models.WafMetricsResponseSeriesItemData"
- "com.azure.resourcemanager.cdn.models.WafMetricsResponseSeriesPropertiesItemsItem"
- "com.azure.resourcemanager.cdn.models.WafRankingGroupBy"
- "com.azure.resourcemanager.cdn.models.WafRankingType"
- "com.azure.resourcemanager.cdn.models.WafRankingsResponseDataItem"
- "com.azure.resourcemanager.cdn.models.WafRankingsResponseDataItemMetric"
- "com.azure.resourcemanager.cdn.models.WafRuleType"
enums:
- "com.azure.resourcemanager.cdn.models.AfdMinimumTlsVersion"
- "com.azure.resourcemanager.cdn.models.GeoFilterActions"
- "com.azure.resourcemanager.cdn.models.HealthProbeRequestType"
- "com.azure.resourcemanager.cdn.models.MinimumTlsVersion"
- "com.azure.resourcemanager.cdn.models.ProbeProtocol"
- "com.azure.resourcemanager.cdn.models.QueryStringCachingBehavior"
- "com.azure.resourcemanager.cdn.models.ResponseBasedDetectedErrorTypes"
- "com.azure.resourcemanager.cdn.models.SharedPrivateLinkResourceStatus"
- "com.azure.resourcemanager.cdn.models.WafMetricsResponseSeriesItemUnit"
interfaces:
- "com.azure.resourcemanager.cdn.models.CdnEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages.AttachablePremium"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages.AttachableStandard"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages.Blank"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages.Blank.PremiumEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages.Blank.StandardEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages.WithPremiumAttach"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.DefinitionStages.WithStandardAttach"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.Update"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages.AttachablePremium"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages.AttachableStandard"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages.Blank"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages.Blank.PremiumEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages.Blank.StandardEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages.WithPremiumAttach"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateDefinitionStages.WithStandardAttach"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdatePremiumEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnEndpoint.UpdateStandardEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnProfile"
- "com.azure.resourcemanager.cdn.models.CdnProfile.Definition"
- "com.azure.resourcemanager.cdn.models.CdnProfile.DefinitionStages"
- "com.azure.resourcemanager.cdn.models.CdnProfile.DefinitionStages.Blank"
- "com.azure.resourcemanager.cdn.models.CdnProfile.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.cdn.models.CdnProfile.DefinitionStages.WithGroup"
- "com.azure.resourcemanager.cdn.models.CdnProfile.DefinitionStages.WithPremiumVerizonCreate"
- "com.azure.resourcemanager.cdn.models.CdnProfile.DefinitionStages.WithSku"
- "com.azure.resourcemanager.cdn.models.CdnProfile.DefinitionStages.WithStandardCreate"
- "com.azure.resourcemanager.cdn.models.CdnProfile.Update"
- "com.azure.resourcemanager.cdn.models.CdnProfile.UpdateStages"
- "com.azure.resourcemanager.cdn.models.CdnProfile.UpdateStages.WithEndpoint"
- "com.azure.resourcemanager.cdn.models.CdnProfiles"
desc: "Package containing the data models for CdnManagementClient. Cdn Management Client."
metadata: {}
package: "com.azure.resourcemanager.cdn.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-cdn:2.19.0
