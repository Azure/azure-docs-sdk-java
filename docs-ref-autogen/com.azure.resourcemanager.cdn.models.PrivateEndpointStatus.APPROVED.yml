### YamlMime:JavaMember
uid: "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.APPROVED*"
fullName: "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.APPROVED"
  fullName: "com.azure.resourcemanager.cdn.models.PrivateEndpointStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointStatus."
  syntax: "public static final PrivateEndpointStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.cdn.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-cdn:2.1.0
